core-bundle/contao/themes/flexible/styles/main.css

```
@@ -2632,6 +2641,14 @@ ul.sortable button[disabled]:hover {
 	background: rgba(255,255,255,.7);
 }
 
+.sortable-chosen {
+	opacity: 0.6 !important;
+}
+
+.sortable-drag {
+	opacity: 0 !important;
```

```suggestion
	opacity: 0.6;
}

.sortable-drag {
	opacity: 0;
```
From testing all widgets there is no need to add the `!important` here?